src/heartbeat.ts

```diff
@@ -12,11 +12,24 @@ import { ITelemetryService } from './services/telemetryService';
 import { SSHConnectionParams } from './remote';
 import { ISessionService } from './services/sessionService';
 import { ILogService } from './services/logService';
+import { RawTelemetryEventProperties } from './common/telemetry';
+
+const IDEHeartbeatTelemetryEvent = 'ide_heartbeat';
+interface IDEHeartbeatTelemetryData extends RawTelemetryEventProperties {
+    clientKind: 'vscode-desktop'; // 'ssh' | 'jetbrains' | 'vscode-desktop' | 'supervisor-frontend';
+    totalCount: number;
+    successfulCount: number;
+    workspaceId: string;
+    instanceId: string;
+    gitpodHost: string;
+    debugWorkspace: 'true' | 'false';
```

Comment:
Can this be a boolean to be consistent with `connectionInfo.debugWorkspace`?

Reply:
> Can this be a boolean to be consistent with connectionInfo.debugWorkspace?

It can be, but I just want to keep it like original field, I guess maybe it's because Mixpanel can't properly identify `undefined` and `false`